### ISSUE TITLE
Get Amplify Admin UI in existing cli project

### ISSUE THREAD

**Issue by davidbiller** [aws-amplify/amplify-cli#6083]
Hey, in my project in the amplify console I can't find any backend information. 

So I have 2 env. develop and production. 
I use the cli to deploy my changes. 

Now there is the new Admin UI, but when I check the Amplify console, I can see this: 

![Bildschirmfoto 2020-12-06 um 14 26 07](https://user-images.githubusercontent.com/7137361/101281407-fd749a00-37ce-11eb-93f8-1b516ef4c5b3.png)

So at first I have to add my backend to this app in console. How can I do this? 

thanks for help

**Comment by kaustavghosh06** [comment]
@davidbiller These steps out here - https://docs.amplify.aws/console/adminui/start#to-get-started-with-a-new-amplify-app should help you get started with Admin UI for existing apps.

**Comment by davidbiller** [comment]
@kaustavghosh06 
I guess this was to fast.

I already have a backend. Created with the cli Long Time ago.

But I can’t choose this environments in the console. 

Also, i can’t pull the amplify backend project in other Frontend apps, because there is no backend connected. This is why I can’t use the admin ui I guess

**Comment by kaustavghosh06** [comment]
@davidbiller Have you between 2019 Nov and today upgraded the CLI and run a push on an existing environment? That should ideally create an amplify app and a corresponding backend environment in the console.

**Comment by davidbiller** [comment]
I can pull the backend in the main Frontend app with the cli. 
When I go to the console 

> @davidbiller Have you between 2019 Nov and today upgraded the CLI and run a push on an existing environment? That should ideally create an amplify app and a corresponding backend environment in the console.

Yes. And, it was so stupid, but I deleted this app after that, because I thought, this was a Test project, because I didn’t created it by myself. 

So after I deleted, I created it by hand again

@kaustavghosh06

**Comment by davidbiller** [comment]
@kaustavghosh06 when I delete the console app, and push again, maybe the cli will create again the project? 

Would be ok, so I have to connect again the domains, but maybe it’s possible in the night.

**Comment by kaustavghosh06** [comment]
You don't need to create it by hand in the console. If you're using the CLI, it should automatically create an Amplify app in the console and have the corresponding backend-environments out there. Do you see a corresponding app in the Amplify console anywhere tied to your CLI project? You can find the appId in the `amplify/backend/amplify-meta.json` file.

**Comment by davidbiller** [comment]
> You don't need to create it by hand in the console. If you're using the CLI, it should automatically create an Amplify app in the console and have the corresponding backend-environments out there. Do you see a corresponding app in the Amplify console anywhere tied to your CLI project? You can find the appId in the `amplify/backend/amplify-meta.json` file.

I only see my own connected app. 
I added this over codeCommit.

**Comment by kaustavghosh06** [comment]
Ah, so you deleted your app from the Amplify Console UI which the CLI created?

**Comment by davidbiller** [comment]
> Ah, so you deleted your app from the Amplify Console UI which the CLI created?

Correct. It was to late after I checked that this was the correct app in console.

**Comment by kaustavghosh06** [comment]
I'll need to check with the team if we can recover a deleted amplify console app and get back to you.

**Comment by kaustavghosh06** [comment]
@davidbiller In the meanwhile could you also provide the appId for the deleted app? 
You can find a reference to it within your local `amplify/backend/amplify-meta.json` file.

**Comment by davidbiller** [comment]
<img width="271" alt="Bildschirmfoto 2020-12-07 um 08 10 09" src="https://user-images.githubusercontent.com/7137361/101320213-a1a82080-3863-11eb-8c52-4c166550714d.png">
@kaustavghosh06 don't have this file?

**Comment by davidbiller** [comment]
why I don't have the amplify-meta.json?
but found the info here: amplify/team-provider-info.json


"StackId": "arn:aws:cloudformation:eu-central-1:327300291334:stack/aivy-rn-app-production-20190802143852/7d9baa60-b522-11e9-a97b-021298c8e4cc",

"AmplifyAppId": "d3vzlf3ypw8gx2"

**Comment by kaustavghosh06** [comment]
@davidbiller Does your VSCode setting hide files which are put in .gitignore?

**Comment by davidbiller** [comment]
> @davidbiller Does your VSCode setting hide files which are put in .gitignore?

ahhh :D sorry. yes. 

"StackName": "aivy-rn-app-develop-20190803185319",
      "StackId": "arn:aws:cloudformation:eu-central-1:327300291334:stack/aivy-rn-app-develop-20190803185319/329a52d0-b60f-11e9-a42b-0ac843449136",
 "AmplifyAppId": "d341qnr5lndklc"

this is the **develop** env.

**Comment by kaustavghosh06** [comment]
As a workaround, if you want to create a new Amplify App ID, you might want to just remove "AmplifyAppId" references in your `amplify-meta.json` and `team-provider.json` file and that should create a new amplify app with a new appId in the Amplify Console for you on a next `amplify push`.

**Comment by davidbiller** [comment]
Okay. Thank you! So I connected the (by my own created console app) app with cloudfront, but I guess it is possible to delete the domain connection and add the domain than in the new app created by the cli. A short offline time would be ok.

**Comment by davidbiller** [comment]
@kaustavghosh06 
I tried the workaround. I deleted the complete line
`"AmplifyAppId": "d341qnr5lndklc"`
in both files. 

After push, there is no new app. and the lines are not there again?

**Comment by davidbiller** [comment]
> As a workaround, if you want to create a new Amplify App ID, you might want to just remove "AmplifyAppId" references in your `amplify-meta.json` and `team-provider.json` file and that should create a new amplify app with a new appId in the Amplify Console for you on a next `amplify push`.

@kaustavghosh06 
I tried the workaround. I deleted the complete line
"AmplifyAppId": "d341qnr5lndklc"
in both files.

After push, there is no new app. and the lines are not there again?
